helmfile/charts/grafana-ops/dashboards/trivy-operator-dashboard.json

```diff
@@ -67,39 +24,171 @@
   "description": "This Dashboard is used to visualise the metrics from the security reports of the Trivy Operator",
   "editable": true,
   "fiscalYearStartMonth": 0,
+  "gnetId": 17813,
   "graphTooltip": 0,
-  "id": null,
   "links": [],
   "liveNow": false,
   "panels": [
     {
-      "collapsed": true,
+      "collapsed": false,
       "gridPos": {
         "h": 1,
         "w": 24,
         "x": 0,
         "y": 0
       },
       "id": 12,
+      "panels": [],
+      "title": "Quick Overview",
+      "type": "row"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "${datasource}"
+      },
+      "description": "Total number of security issues by type.\nNote that issues related to images may be duplicated when running in multiple namespaces and workloads.",
+      "fieldConfig": {
+        "defaults": {
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "dark-green",
+                "value": null
+              },
+              {
+                "color": "dark-red",
+                "value": 1
+              }
+            ]
+          },
+          "unit": "none"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 5,
+        "w": 24,
+        "x": 0,
+        "y": 1
+      },
+      "id": 31,
+      "options": {
+        "colorMode": "background",
+        "graphMode": "none",
+        "justifyMode": "center",
+        "orientation": "vertical",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "textMode": "value_and_name"
+      },
+      "pluginVersion": "9.3.13",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "sum(max(trivy_image_exposedsecrets{cluster=~\"$cluster\", severity=~\"$severity\", namespace=~\"$namespace\"}) by (image_repository, image_tag, severity))",
+          "instant": true,
+          "legendFormat": "Image Exposed Secrets",
+          "range": false,
+          "refId": "A"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "sum(max(trivy_image_vulnerabilities{cluster=~\"$cluster\", severity=~\"$severity\", namespace=~\"$namespace\"}) by (image_repository, image_tag, severity))",
```

Comment:
I see you have this in several queries and I think this it is not needed.
```suggestion
          "expr": "sum(max(trivy_image_vulnerabilities{cluster=~\"$cluster\", severity=~\"$severity\", namespace=~\"$namespace\"}) by (image_repository, image_tag))",
```

Reply:
It is needed because it is how I filter out the results to be unique over images.
So I do `max by (image_repository, image_tag, severity)` then work with them further.